app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -239,6 +243,27 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                         .zoom(ZOOM_LEVEL)
                         .build();
                 mapBoxMap.setCameraPosition(cameraPosition);
+                scaleBarPlugin = new ScaleBarPlugin(mapView, mapBoxMap);
+
+                int color = isDarkTheme?R.color.bottom_bar_light:R.color.bottom_bar_dark;
```

Comment:
This spacing is not in style for the project, try pressing `ctrl+alt+l` in this file, or consider setting up a macro like mentioned in the [docs](https://github.com/commons-app/documentation/blob/master/android/Code-style.md)

Reply:
@318anushka this is not updated

Reply:
@macgills  this formats the entire class.
![image](https://user-images.githubusercontent.com/45694892/76754746-9fb7a200-67a8-11ea-9ea0-7d4d37219bb9.png)

Reply:
I am okay with the entire class being formatted but I would defer to @misaochan here.

You other option is selecting the lines you have changed and then formatting

Reply:
There isn't much difference but here it is
![image](https://user-images.githubusercontent.com/45694892/76755558-14d7a700-67aa-11ea-91a1-cd2176a0e3ef.png)

Reply:
Changing those lines specifically is good. Please don't format the entire class in this PR. :) You can submit a separate PR for the class's reformatting.

Reply:
Okay!!
So @macgills should i push final changes? It's the image right above after reformatting my lines.

Reply:
move the comments so they are above the variables they are documenting to stop the breaks looking so weird & hard to read and then push away.

Reply:
Done. Please check 😊